discord/no-decay/index.html

```diff
@@ -486,11 +422,9 @@ <h1 class="title is-1">Meet the staff<i>!</i></h1>
                                                                     </div>
                                                                 </div>
                                                                 <div class="content">
-                                                                    <p class="subtitle is-6 has-text-dark"><b>SR:</b> 2300</p>
                                                                     <p>
-                                                                        Manages Team Ram3n and owns Team Rh1no. Best main support on Team T2raining Bots and shot calls for Team T2raining Bots.
```

Comment:
This was a joke because she is the only main support. 😛

Reply:
haha! Yeah I get it's a joke! but no one else is "best" at their role ;)